Sources/POSIX/chdir.swift

```diff
@@ -16,7 +16,7 @@ import var libc.errno
 */
 public func chdir(_ path: String) throws {
     if memo == nil {
-        let argv0 = try realpath(CommandLine.arguments.first!)
+        let argv0 = try realpath(executable: CommandLine.arguments.first!)
```

Comment:
If this is compiled into dylib, this won't work properly. We may need to rethink the implementation of this API.

Reply:
How do you compile the project into dylib?

Reply:
If your project is an executable, it will work fine. But if its a dynamic library product, it will break for its users.